doc/articles/get-started.md

```diff
@@ -8,13 +8,12 @@ To set up your development environment, first select the operating system you're
 
 # [**Windows**](#tab/windows)
 
-If you're developing on Windows, we recommend you use **Visual Studio 2022**, for the richest XAML development experience and broadest platform coverage. 
+If you're developing on Windows, we recommend you use [**Visual Studio 2022**](get-started-vs-2022.md), for the richest XAML development experience and broadest platform coverage. 
 
 If you already use and love **JetBrains Rider** or **Visual Studio Code**, you can also use them to develop Uno Platform applications. Check the support matrix below to see which target platforms they support.
 
 **Choose the IDE you want to use:**
 
- - [Get started with Visual Studio 2022 for Windows](get-started-vs-2022.md)
```

Comment:
I would keep the bullet point here for completeness anyway (if you scan the page quickly, it would seem that there are only two options this way)

Reply:
noted, I think it might be confusing/redundant to have it twice - will leave as is for now and confirm with Sasha